pyproject.toml

```diff
@@ -19,17 +17,18 @@ classifiers = [
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
 ]
 packages = [{ include = "appmail" }]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-django = "^3.0 || ^4.0"
+django = "^3.2 || ^4.0"
 
 [tool.poetry.dev-dependencies]
-black = {version = "*", allow-prereleases = true}
+black = "*"
```

Comment:
This is old - Black now has a released version so don't need to specify pre-release.